workspace/__lib__/xod-dev/pn532-nfc/example-read-page/patch.xodp

```diff
@@ -0,0 +1,307 @@
+{
+  "comments": [
+    {
+      "content": "<- Create the device",
+      "id": "BJNQ-vLum",
+      "position": {
+        "x": 340,
+        "y": 0
+      },
+      "size": {
+        "height": 51,
+        "width": 306
+      }
+    },
+    {
+      "content": "<- Read data from one page",
+      "id": "ByOhbv8dX",
+      "position": {
+        "x": 340,
+        "y": 510
+      },
+      "size": {
+        "height": 51,
+        "width": 306
+      }
+    },
+    {
+      "content": "<- Initialize it on boot",
+      "id": "ByzV-vLdQ",
+      "position": {
+        "x": 340,
+        "y": 102
+      },
+      "size": {
+        "height": 51,
+        "width": 306
+      }
+    },
+    {
+      "content": "<- Detect an NFC tag and read its UID. This is a required step to read or write data to the tag",
+      "id": "H185bw8_X",
+      "position": {
+        "x": 340,
+        "y": 408
+      },
+      "size": {
+        "height": 51,
+        "width": 306
+      }
+    },
+    {
+      "content": "<- After initialization it will pulse continuously to trigger detecting of an NFC tag on each transaction",
```

Comment:
IMHO, this is way too technical. Do all users have to know about execution model and what a transaction is?